org.flathub.WineBaseApp.yml

```diff
@@ -0,0 +1,142 @@
+id: org.flathub.WineBaseApp
```

Comment:
Should it be renamed to something shorter, e.g. `org.flathub.WineBase` (or even `org.flathub.WinePak`)?

Reply:
WinePak can cause confusion because it would seem that it is related to winepak.org

Reply:
Technically speaking it's `WineDepsBaseApp`, no?

Reply:
As there are no `ElectronDepsBaseApp` or `FirefoxDepsBaseApp` then I think the `Deps` part would be inconsistent here.